packages/strapi-parts/src/Box/Box.stories.mdx

```diff
@@ -32,3 +41,10 @@ Description...
     </Box>
   </Story>
 </Canvas>
+
+
+## Props
+
+<!-- TODO: args not loading here -->
+
```

Comment:
not sure why but the args are not being displayed here.

Reply:
This is because it is a styled component.

You can change the Box.js file as follows

```
import React from 'react';

const BaseBox = styled.div.withConfig({...});

const Box = ({ children, ...props }) => <BaseBox {...prop}>{children}</BaseBox>;

Box.defaultProps = {
 // ...,
 children: undefined,
};

Box.propTypes = {
 children: PropTypes.node
 // ...
};
```